integration/bundle/bind_resource_test.go

```diff
@@ -15,8 +15,62 @@ import (
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
 )
 
+func TestBindSchemaToExistingSchema(t *testing.T) {
+	ctx, wt := acc.UcWorkspaceTest(t)
+
+	// create a pre-defined schema:
+	uniqueId := uuid.New().String()
+	predefinedSchema, err := wt.W.Schemas.Create(ctx, catalog.CreateSchema{
+		CatalogName: "main",
+		Name:        "test-schema-" + uniqueId,
+	})
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		err := wt.W.Schemas.DeleteByFullName(ctx, predefinedSchema.FullName)
+		require.NoError(t, err)
+	})
+
+	// setup the bundle:
+	bundleRoot := initTestTemplate(t, ctx, "uc_schema", map[string]any{
```

Comment:
Does this template do more than just the schema?

If so it makes sense to create a dedicated template for this test so it is isolated.